### ISSUE TITLE
Missing export buttons

### ISSUE THREAD

**Issue by tomas-knap** [odalic/odalic-ui#350]
When on full screen, no export buttons in Step 1/2/3
When resized to smaller screen they appeared

**Comment by tomas-knap** [comment]
With https://github.com/odalic/odalic-ui/pull/355/commits/1057f943fbb70743fdf4bf9c04cb060e1c20e57c it works for me.

But the buttons are strangely small: 

Before (Odalic v0.4):

![screen shot 2017-02-23 at 10 59 07](https://cloud.githubusercontent.com/assets/3014917/23254109/2e8f6cd2-f9b7-11e6-9aa2-ab175a679fcd.png)

Now: 

![screen shot 2017-02-23 at 10 59 50](https://cloud.githubusercontent.com/assets/3014917/23254123/3d7e01ea-f9b7-11e6-8ca0-e8c2c8366450.png)

Also there is very awful space after the table and before Exports, which was not there before

**Comment by istvan-satmari** [comment]
erm, that is something I caused, but there is a reason for that - you see, buttons are separate elements, but to be able to download resources via AJAX, we had to do some HTML5 tricks and wrapped the code around "afake" directive; afake is "<a ...>" type of element. So they are kind of incompatible. But wrapping buttons around <a>...</a> and clicking on them still counts as clicking on the hyperlink, so... yeah, this happened.

I propose not to have buttons, but rather clickable text, styled like a button. That should solve the issue.
